tests/cross/src/main/scala/tests/BasePCSuite.scala

```diff
@@ -1,105 +1,151 @@
 package tests
 
-import coursier._
-import java.nio.charset.StandardCharsets
 import java.nio.file.Files
 import java.nio.file.Path
-import java.nio.file.Paths
+import java.util.concurrent.Executors
+import java.util.concurrent.ScheduledExecutorService
+import java.nio.charset.StandardCharsets
+import coursierapi.Dependency
+import coursierapi.Fetch
 import org.eclipse.lsp4j.MarkupContent
 import org.eclipse.lsp4j.jsonrpc.messages.{Either => JEither}
 import scala.meta.internal.jdk.CollectionConverters._
 import scala.meta.internal.metals.ClasspathSearch
 import scala.meta.internal.metals.JdkSources
 import scala.meta.internal.metals.Docstrings
 import scala.meta.internal.metals.RecursivelyDelete
-import scala.meta.internal.mtags.BuildInfo
-import scala.meta.internal.mtags.OnDemandSymbolIndex
-import scala.meta.internal.mtags.ClasspathLoader
 import scala.meta.internal.pc.PresentationCompilerConfigImpl
-import scala.meta.internal.pc.ScalaPresentationCompiler
-import scala.meta.internal.metals.PackageIndex
+import scala.meta.internal.mtags.GlobalSymbolIndex
 import scala.meta.io.AbsolutePath
 import scala.meta.pc.PresentationCompilerConfig
-import scala.util.control.NonFatal
-import java.util.concurrent.Executors
-import java.util.concurrent.ScheduledExecutorService
 import scala.collection.Seq
+import scala.util.control.NonFatal
 import scala.meta.pc.PresentationCompiler
+import scala.meta.internal.pc.ScalaPresentationCompiler
+import scala.meta.internal.metals.PackageIndex
+import munit.Tag
 
 abstract class BasePCSuite extends BaseSuite {
-  def thisClasspath: Seq[Path] =
```

Comment:
The technique of inspecting `this.getClass.getClassLoader` to extract scala-library.jar works when you run `+cross/test`. It's significantly faster compared to using `coursier.Fetch.create()`.